src/v3/src/transformer/layout/idp/transformIdvIdpAuthenticator.ts

```diff
@@ -58,7 +94,36 @@ export const transformIdvIdpAuthenticator: IdxStepTransformer = ({
     },
   };
 
-  uischema.elements = [titleElement, descriptionElement, submitButton];
+  const divider: DividerElement = {
+    type: 'Divider',
+  };
+
+  uischema.elements = [
+    titleElement,
+    descriptionElement,
+    submitButton,
+    divider,
+  ];
+
+  if (shouldShowCancelLink(widgetProps?.features)) {
+    const cancelLink: LinkElement = {
+      type: 'Link',
+      contentType: 'footer',
+      options: {
+        label: loc('goback', 'login'),
+        isActionStep: true,
+        step: 'cancel',
+      },
```

Comment:
Let's also add in `dataSe: 'cancel'` here to be consistent with [transformCancelButton](https://github.com/okta/okta-signin-widget/blob/8c660a4277d5113e81efe9c9a70cdeaedde32955/src/v3/src/transformer/button/transformCancelButton.ts#L45)